github/ci/services/grafana/hack/test.sh

```diff
@@ -7,36 +7,23 @@ PROJECT_INFRA_ROOT=$(readlink --canonicalize ${BASEDIR}/../../../../../)
 
 cd $PROJECT_INFRA_ROOT
 
-cat <<EOF | kind create cluster --image quay.io/kubevirtci/kindest-node:v1.27.1 --config=-
-kind: Cluster
-apiVersion: kind.x-k8s.io/v1alpha4
-nodes:
-- role: control-plane
-  kubeadmConfigPatches:
-  - |
-    kind: InitConfiguration
-    nodeRegistration:
-      kubeletExtraArgs:
-        node-labels: "ingress-ready=true"
-  extraPortMappings:
-  - containerPort: 80
-    hostPort: 80
-    protocol: TCP
-  - containerPort: 443
-    hostPort: 443
-    protocol: TCP
-EOF
+KUBEVIRT_DIR=${KUBEVIRT_DIR:-/home/prow/go/src/github.com/kubevirt/kubevirt}
 
-kubectl create ns monitoring
+export KUBEVIRT_PROVIDER=k8s-1.28
```

Comment:
Is there a reason why we don't use the default provider?

Reply:
Yes lets use the default - that was a leftover from something else I was trying.